tools/docker/tests/flows/__init__.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python3
+import os
+import unittest
+import re
+from enum import Enum
+import sys
+
+class DeviceType(Enum):
+    gateway = 1
+    agent = 2
+    repeater1 = 3
+    repeater2 = 4
+
+class LogType(Enum):
+    beerocks_agent = 1
+    beerocks_agent_wlan0 = 2
+    beerocks_agent_wlan2 = 3
+    beerocks_monitor_wlan0 = 4
+    beerocks_monitor_wlan2 = 5
+    beerocks_controller = 6
+    transport = 7
+    local_bus = 8
+
+class BaseTest(unittest.TestCase):
+
+    def send_controller_command(self, command):
+        testSystem.send_command(Device.gateway, command)
```

Comment:
I don't think these wrappers add value. You can just as well call this directly from the test.